src/dodal/devices/hutch_shutter.py

```diff
@@ -0,0 +1,85 @@
+from enum import Enum
+
+from bluesky.protocols import Movable
+from ophyd_async.core import (
+    DEFAULT_TIMEOUT,
+    AsyncStatus,
+    StandardReadable,
+    wait_for_value,
+)
+from ophyd_async.epics.signal import epics_signal_r, epics_signal_w
+
+from dodal.log import LOGGER
+
+
+class HutchNotInterlockedError(Exception):
+    def __init__(self, errmsg, *args: object) -> None:
+        LOGGER.error(errmsg)
+        super().__init__(*args)
+
+
+class ShutterDemand(str, Enum):
+    OPEN = "Open"
+    CLOSE = "Close"
+    RESET = "Reset"
+
+
+class ShutterState(str, Enum):
+    FAULT = "Fault"
+    OPEN = "Open"
+    OPENING = "Opening"
+    CLOSED = "Closed"
+    CLOSING = "Closing"
+
+
+class HutchInterlock(StandardReadable):
+    """Device to check the interlock status of the hutch."""
+
+    def __init__(self, bl_prefix: str, name: str = "") -> None:
+        self.status = epics_signal_r(int, bl_prefix + "-PS-IOC-01:M14:LOP")
+        super().__init__(name)
+
+    async def is_insterlocked(self) -> bool:
```

Comment:
Should: 
```suggestion
    async def is_interlocked(self) -> bool:
```

Reply:
Should: I think the wording of "is interlocked" is a bit unclear. If you told me that something was interlocked I wouldn't be 100% sure if this meant it was free to move or not. Maybe better would be to call it all "safe_to_move". We could also put the 0 used below into a constant of "safe to move"

Reply:
I agree that interlock is not very clear. I reworded this to "shutter_safe_to_operate".